website/src/parsers/js/transformers/babel7/index.js

```diff
@@ -28,32 +30,7 @@ export default {
     return babel.transformAsync(code, {
       parserOpts: {
         parser: recast.parse,
-        plugins: [
-          'asyncGenerators',
-          'bigInt',
-          'classPrivateMethods',
-          'classPrivateProperties',
-          'classProperties',
-          ['decorators', {decoratorsBeforeExport: false}],
-          'doExpressions',
-          'dynamicImport',
-          'exportDefaultFrom',
-          'exportNamespaceFrom',
-          'flow',
-          'flowComments',
-          'functionBind',
-          'functionSent',
-          'importMeta',
-          'jsx',
-          'logicalAssignment',
-          'nullishCoalescingOperator',
-          'numericSeparator',
-          'objectRestSpread',
-          'optionalCatchBinding',
-          'optionalChaining',
-          ['pipelineOperator', {proposal: 'minimal'}],
-          'throwExpressions',
-        ],
```

Comment:
Not sure if this will break things, but seems reasonable that this list should be the same as the input/parser.